src/Draft/FirstDraft.tsx

```diff
@@ -0,0 +1,268 @@
+// /* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Delete this file

Reply:
> Delete this file

Not fixed
Also, as I see that all code is commented here, so remove it and all comments in other places

Reply:
still not fixed